core/src/test/scala/io/finch/response/ResponseBuilderSpec.scala

```diff
@@ -1,71 +1,98 @@
-/*

```

Comment:
Not sure what happened with this file, but it looks like a problem with either tabs of new lines. @yanana are you on Mac?